susemanager-utils/susemanager-sls/salt/cleanup_minion/init.sls

```diff
@@ -9,3 +9,8 @@ mgrchannels_repo_clean_all:
     - name: rm "/etc/zypp/repos.d/susemanager:channels.repo"
     - runas: root
 {%- endif %}
+{%- if grains['os_family'] == 'Debian' %}
+  cmd.run:
+    - name: rm "/etc/apt/sources.list.d/susemanager:channels.list" "/usr/share/keyrings/mgr-archive-keyring.gpg"
+    - runas: root
```

Comment:
This state return error when metadata signing is not enabled, therefore the keyring file is not there.

I think we can use `file.absent` state here also for a general improvement.

Reply:
Sure. Sounds good. I'm just confused, that I named the file ```susemanager:channels.list```